client/common/common.c

```diff
@@ -176,7 +176,7 @@ usage(FILE *out, const char *name)
           " -p, --prompt          defines the prompt text to be displayed.\n"
           " -P, --prefix          text to show before highlighted item.\n"
           " -I, --index           select item at index automatically.\n"
-          " -x, --password        hide input.\n"
+          " -x, --password        display/hide/replace input. (none(default), hide(no displayed text), indicator(replace with asterisks)\n"
```

Comment:
add space `none (default)`, I don't think the other options need parenthesis description. Also edit the man page

Reply:
Where can I edit the man page? "man bemenu" or "man bemenu-run" doesn't seem to work for me.

Reply:
https://github.com/Cloudef/bemenu/blob/master/man/bemenu.1.scd.in

Reply:
Does the man page have to be built? Is my problem a common issue?

Reply:
Ah, sorry. I didn't see the scdoc mention in the README.md.